stylesheets/_component.navigation.scss

```diff
@@ -179,8 +179,9 @@
     border-radius: 4px;
     color: $nav-color-dark;
     display: inline-block;
+    height: 34px;
```

Comment:
Is there a need to set height on a button here?

Reply:
I was struggling to set the height of toolbar elements to a consistent height